source/Calamari.AzureAppService/Json/LegacyConnectionStringsSetting.cs

```diff
@@ -1,14 +1,10 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using Microsoft.Azure.Management.WebSites.Models;
-using Newtonsoft.Json;
+using ArmConnectionStringType = Azure.ResourceManager.AppService.Models.ConnectionStringType;
 
 namespace Calamari.AzureAppService.Json
 {
-    public class ConnectionStringSetting
+    public class LegacyConnectionStringSetting
```

Comment:
I needed a legacy version of this because the `ConnectionStringType` enum is a different value... One wonders if we should have our own version of the enum as these values are being parsed out of the variables 🤔